### PR TITLE
module: snp-dists

### PR DESCRIPTION
**Pull request by abhi18av**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Comment by rpetit3** [comment]
@abhi18av  I'm good with it, but think one of the core members should probably make the final decision

**Comment by abhi18av** [comment]
Yeah, makes sense. I've request the `modules-team` to review the PR now 🙏

File: tests/modules/snpdists/test.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml snpdists
```
Comment (grst) [inline tests/modules/snpdists/test.yml:2]: leftover todo statement

**Review by grst — COMMENTED** [review]
looks good, just found a leftover TODO statement

File: tests/modules/snpdists/test.yml (hunk 1 above)
Comment (abhi18av) [reply tests/modules/snpdists/test.yml]: Whoops!

Fixed with https://github.com/nf-core/modules/pull/697/commits/ac28108107fba07809c8c0257cbc6b0bc3826e6b 

Thanks for the review @grst !

**Review by grst — APPROVED** [review]

File: modules/snpdists/main.nf [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    tuple val(meta), path("*.tsv"), emit: tsv
+    path "*.version.txt"          , emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    snp-dists \\
+        $options.args \\
+        ${alignment} > ${prefix}.tsv
```
Comment (drpatelh) [inline modules/snpdists/main.nf:34]:
```suggestion
        $alignment > ${prefix}.tsv
```

**Review by drpatelh — APPROVED** [review]
LGTM too!

**Pull request merged**